drawfunctions/satoshi_s.py

```diff
@@ -0,0 +1,24 @@
+import turtle
+
+
+def base():
+    t = turtle.Turtle()
+    t.color("brown")
+
```

Comment:
Could you please add a `hideturtle` call before starting the drawing? This helps with performance.

```suggestion
    t.hideturtle()
```

Reply:
Updated those changes, @dominicduffin1